xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/extension.vm

```diff
@@ -1016,6 +1016,62 @@ $namespace##
   </div>
 #end
 
+#macro (displayExtensionSupportPlans $extensionSupportPlans $supporterList)
+  #set ($extensionSupporters = $extensionSupportPlans.supporters)
+  #if (!$extensionSupporters.isEmpty())
+      <div class="supporter-listing">
+        #if ($supporterList)
+          #set ($supporterElementName = 'div')
+        #else
+          #set ($supporterElementName = 'span')
+        #end
+        #set ($multi-first = false)
+        #foreach ($extensionSupporter in $extensionSupporters)
+          #set ($extensionSupporterPlans = $extensionSupportPlans.getSupportPlans($extensionSupporter))
+          #if ($extensionSupporterPlans.size() == 1)
+            <$supporterElementName class="supporter support-single-plan">
+              #set ($extensionSupporterPlan = $extensionSupporterPlans.get(0))
+              <span class="supporter-name">$extensionSupporter.name: </span>
+              <span class="plan-name infomessage">
+                <a target="_blank" class="external wikiexternallink" href="$escapetool.xml($extensionSupporterPlan.uRL)">
```

Comment:
I wonder if we should validate the URL using https://github.com/xwiki/xwiki-commons/blob/f369c38d5ddbedba4574e2c536f4c1bc5b1386d8/xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/html/script/HTMLScriptService.java#L73 to avoid, e.g., JavaScript URLs. Further, this needs a `rel="noopener noreferrer"` attribute for security.

Reply:
> I wonder if we should validate the URL using [xwiki/xwiki-commons@f369c38/xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/html/script/HTMLScriptService.java#L73](https://github.com/xwiki/xwiki-commons/blob/f369c38d5ddbedba4574e2c536f4c1bc5b1386d8/xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/html/script/HTMLScriptService.java#L73)

You mean with something like `$services.html.isAttributeSafe('a', 'href', $extensionSupporterPlan.uRL)` ?

Reply:
> You mean with something like `$services.html.isAttributeSafe('a', 'href', $extensionSupporterPlan.uRL)` ?

Yes, exactly. This prevents URLs like `javascript:alert('XSS');`.